cmd/operator/main.go

```diff
@@ -62,33 +62,12 @@ func main() {
 		publicNamespace = flag.String("public-namespace", operator.DefaultPublicNamespace,
 			"Namespace in which the operator reads user-provided resources.")
 
-		imageCollector = flag.String("image-collector", operator.ImageCollector,
```

Comment:
Are you worried we'll be breaking any customers' existing setups by removing these flags? Obviously wouldn't be an issue on GKE clusters using the gcloud integration.

I guess those who are deploying via `kubectl` themselves would use our manifests as a guide. But I'm sure they adapt those for their specific cases.

Do you think being loud enough in our release notes about the flag removals is sufficient?

Reply:
I think so. The operator is in v0 and that primarily covers flags as our CRD API is versioned seperately. So users should always read release notes.

For upgrading, one interesting point is probably that the operator may override the YAML manifests again before being upgraded itself. So some extra caution is probably in order for kubectl-installed setups.